CONTRIBUTING.md

```diff
@@ -189,6 +189,11 @@ cuda-gdb -ex r --args python <program_name>.py <program_arguments>
 cuda-memcheck python <program_name>.py <program_arguments>
 ```
 
+### Building and Testing on a gpuCI image locally
+
+Before submitting a pull request, you can do a local build and test on your machine that mimics our gpuCI environment using the `ci/local/build.sh` script.
+For detailed information on usage of this script, see [here](https://github.com/rapidsai/cudf/tree/branch-0.7/ci/local).
```

Comment:
Can we do proper documentation rather than linking to a folder (presumably it's a folder since it has no file extension, but the file is 404 right now so can't verify). I don't like linking to specific branches either.

Reply:
I had it linking to the folder since that folder has the README in it. Didn't know you could do relative paths! Updated so it links directly to the README and gave it a relative path.